plugin/evm/api.go

```diff
@@ -0,0 +1,38 @@
+// (c) 2019-2020, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
```

Comment:
should we leave this in `service.go` and add the new stuff in `service_validator.go`?

or we can rename this to something like `api_snowman.go` and have `api_validator.go`?

Reply:
My intention was to keep avalancheRPC server handlers separate from `/rpc` handler, and follow avalanche endpoints -> service, /rpc endpoint -> api convention.